sys-apps/baselayout/baselayout-2.2-r1.ebuild

```diff
@@ -181,7 +181,7 @@ src_prepare() {
 	echo "LDPATH='${ldpaths#:}'" >> etc/env.d/00basic
 
 	# rc-scripts version for testing of features that *should* be present
-	echo "Gentoo Base System release ${PV}" > etc/gentoo-release
+	echo "${DISTRO} Base System release ${PV}" > etc/gentoo-release
```

Comment:
Understandable, and agreed.  We have [adelie-base](https://code.foxkit.us/adelie/packages/tree/master/sys-apps/adelie-base) and add sys-apps/baselayout to package.provided in Adélie; I will undo this change.